docs/contributing-to-airbyte/building-new-connector/standard-source-tests.md

```diff
@@ -2,31 +2,3 @@
 
 Test methods start with `test`. Other methods are internal helpers in the java class implementing the test suite.
 
-## testGetSpec
```

Comment:
seems accidentally deleted? 😜

Reply:
not sure why.. what do I need to do for that?

Reply:
easiest thing is probabbly copy this file from master into your branch and commit it